docs/book/src/user/quick-start.md

```diff
@@ -436,7 +436,65 @@ clusterctl init --infrastructure ibmcloud
 {{#/tab }}
 {{#tab Kubevirt}}
 
-Please visit the [Kubevirt project][Kubevirt provider].
+Please visit the [Kubevirt project][Kubevirt provider] for more information.
+
+KubeVirt is a cloud native virtualization solution. The meaning for that for our guest cluster, is that the virtual
+machines will be created within the kind cluster, as a kubernetes resources. To make it work, we'll need to add some
+configurations to our kind cluster.
+
+##### Install the calico CNI
+The default kind's CNI (kindnet) will not work for the guest cluster. We'll use calico CNI. Download the calico manifest and use it to create the required resources, as described [here](https://projectcalico.docs.tigera.io/getting-started/kubernetes/self-managed-onprem/onpremises#install-calico); e.g.:
+```bash
+curl https://raw.githubusercontent.com/projectcalico/calico/v3.24.3/manifests/calico.yaml -O
+kubectl create -f calico.yaml
```

Comment:
Does this mean that after that kindnet & Calico are installed? Isn't that a problem?

Reply:
> Does this mean that after that kindnet & Calico are installed? Isn't that a problem?

This is a good question, @sbueringer. So in my environment it's working (tested over and over again). It is possible to disable kindnet (https://kind.sigs.k8s.io/docs/user/configuration/#disable-default-cni) but I thought it will make the document a bit too complicated, because I'll need a custom kind cluster creation for KubeVirt, and since it does work without this modification, I thought it will be simpler to leave it as is. WDYT?